src/modules/autoAcceptPendingRelationships/AutoAcceptPendingRelationships.ts

```diff
@@ -1,11 +1,9 @@
 import { RelationshipChangedEvent, RelationshipStatus } from "@nmshd/runtime";
 import { ConnectorRuntimeModule, ConnectorRuntimeModuleConfiguration } from "../../ConnectorRuntimeModule";
 
-export interface AutoAcceptRelationshipCreationChangesModuleConfiguration extends ConnectorRuntimeModuleConfiguration {
-    responseContent: any;
-}
+export interface AutoAcceptPendingRelationshipsModuleConfiguration extends ConnectorRuntimeModuleConfiguration {}
 
-export default class AutoAcceptRelationshipCreationChangesModule extends ConnectorRuntimeModule<AutoAcceptRelationshipCreationChangesModuleConfiguration> {
+export default class AutoAcceptPendingRelationshipsModule extends ConnectorRuntimeModule<AutoAcceptPendingRelationshipsModuleConfiguration> {
```

Comment:
I'm missing the word Incoming here. wtb `AutoAcceptIncomingPendingRelationshipsModule`

Reply:
@tnotheis what's your opinion?

Reply:
I've checked the documentation's "Establish Relationships" page: the word incoming is only used with incoming relationshipRequests, and it will be gone alongside them

Reply:
ok